### PR TITLE
TFLite launcher: unify input parameters

### PR DESCRIPTION
**Pull request by jkamelin**
Updated format for `--input_names`, `--layout` parameters, added support of `input0[value0],input1[value1]` format for `--input_shape`. Removed unused  `--output_names` parameter.

### CODE REVIEW

File: src/inference/inference_tensorflowlite.py [hunk 1]
```diff
@@ -232,31 +204,54 @@ def raw_result_output(average_time, fps, latency):
...
+            raise ValueError(f'Unable to parse input parameter: {values}')
+    return return_values
+
+
+def parse_layout_arg(values, input_names):
+    return_values = {}
+    if values is not None:
+        matches = re.findall(r'(.*?)\((.*?)\),?', values)
+        if matches:
+            for i, match in enumerate(matches):
```
Comment (zmaslova) [inline src/inference/inference_tensorflowlite.py:243]: `i` is not used so you can iterate directly, without enumerate

File: src/inference/inference_tensorflowlite.py [hunk 2]
```diff
@@ -232,31 +204,54 @@ def raw_result_output(average_time, fps, latency):
...
     return dictionary
 
 
-def parse_mean_scale_arg(values, input_names):
+def parse_input_arg(values, input_names):
     return_values = {}
     if values is not None:
         matches = re.findall(r'(.*?)\[(.*?)\],?', values)
         if matches:
             for i, match in enumerate(matches):
```
Comment (zmaslova) [inline src/inference/inference_tensorflowlite.py:223]: same here

**Review by zmaslova — APPROVED** [review]
Approved with minor comments

**Review by just-sparta — APPROVED** [review]

File: src/inference/inference_tensorflowlite.py (hunk 1 above)
Comment (jkamelin) [reply src/inference/inference_tensorflowlite.py:243]: `i` is used when input names are provided separately in `--input_names` parameter. https://github.com/YADRO-KNS/dl-benchmark/pull/6/files/fc9f43619cd2126a66d6e3f18e79e4bb9b69c5c8#diff-1c1d9f4bce6f85f6e29a0f5d57303e1a0da9e4f956feac651c2a61b431bd6e1aL257

Comment (zmaslova) [reply src/inference/inference_tensorflowlite.py:243]: Missed, ok!

File: src/inference/inference_tensorflowlite.py (hunk 2 above)
Comment (zmaslova) [reply src/inference/inference_tensorflowlite.py:223]: Sorry

**Pull request merged**